x11-packages/gtk4/build.sh

```diff
@@ -2,10 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://www.gtk.org/
 TERMUX_PKG_DESCRIPTION="GObject-based multi-platform GUI toolkit"
 TERMUX_PKG_LICENSE="LGPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="4.16.1"
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_VERSION="4.16.2"
 TERMUX_PKG_SRCURL=https://download.gnome.org/sources/gtk/${TERMUX_PKG_VERSION%.*}/gtk-${TERMUX_PKG_VERSION}.tar.xz
```

Comment:
Can't we use the release tags from the GNOME GitLab for the source URL to enable auto updates for GTK?
https://gitlab.gnome.org/GNOME/gtk/-/tags

Might need to specify `4\.\d+\.\d+` for the RegEx to make sure it doesn't try to use the GTK3 releases, but auto updates should be fairly straight forward to do for GTK.

Reply:
I am not sure if auto update should be enabled for gtk4. It follows odd/even rule for versions - odd being unstable/preview release. For example, see the latest stable and preview version in https://gtk.org/

Preview versions of gtk4 can be packaged here but we should not be that much bleeding edge, right?

Reply:
The odd/even version check can be added custom auto-update function in the build script. Please feel free to add that logic in separate PR.